docs/content/docs/2.components/modal.md

```
@@ -360,6 +360,36 @@ slots:
 :placeholder{class="h-full"}
 ::
 
+### Force Mount
+
+Use the `portal` prop with an object to force the Modal content to render even when closed. This is useful for SSR when the modal should be visible on initial page load.
+
+::component-code
+---
+prettier: true
+ignore:
+  - title
```

markdownlint-cli2 (0.20.0)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)